barretenberg/cpp/src/barretenberg/plonk/composer/standard_composer.cpp

```diff
@@ -41,7 +41,7 @@ std::shared_ptr<plonk::proving_key> StandardComposer::compute_proving_key(Circui
         subgroup_size, circuit_constructor.public_inputs.size(), crs, CircuitType::STANDARD);
 
     // Construct and add to proving key the wire, selector and copy constraint polynomials
-    Trace::populate(circuit_constructor, circuit_proving_key);
+    Trace::populate(circuit_constructor, *circuit_proving_key);
```

Comment:
I guess it would have spiraled here to handle not through a pointer here? Seems reasonable to leave as a pointer in that case since Plonk is on its way out.

Reply:
indeed